bot/exts/info/source.py

```diff
@@ -77,18 +90,21 @@ async def build_embed(self, source_object: SourceType) -> Optional[Embed]:
         """Build embed based on source object."""
         url, location, first_line = self.get_source_link(source_object)
 
-        if isinstance(source_object, commands.HelpCommand):
+        if isinstance(source_object, HelpCommand):
             title = "Help Command"
             description = source_object.__doc__.splitlines()[1]
-        elif isinstance(source_object, commands.Command):
+        elif isinstance(source_object, Command):
             description = source_object.short_doc
             title = f"Command: {source_object.qualified_name}"
```

Comment:
If you want, we can change this to
```suggestion
            title = f"Text command: {source_object.qualified_name}"
```

Reply:
Thanks for noticing for that, I didn't notice it.

Reply:
According to discord-dev website they are called `Prefix-based commands`, shall I change it to that or this?

Reply:
I don't think most people would know them under that name, so a text command is better.
Let's see what others have to say about it though

Reply:
I am gonna go with prefix-based commands because the name explains the usage better than text command. As slash commands are also text commands.

Reply:
Updated in https://github.com/python-discord/bot/pull/2446/commits/a1820fa6836d8a0857d93672af9e7738625deafe